### PR TITLE
Run 26 code review

### PR DESCRIPTION
**Pull request by Grisou13**

### CODE REVIEW

File: src/pages/run/run.ts [hunk 1]
```diff
@@ -39,14 +41,17 @@ export class RunPage {
...
-    this.loadRun().subscribe(
-      null,
-      err => err.status != 401 && loader.dismiss(),
-      () => loader.dismiss()
-    )
+    loader.present().then(()=>{
+      this.loadRun().subscribe(
+        ()=>loader.dismissAll(),
+        err => err.status != 401 && loader.dismiss(),
+        // () => loader.dismiss()
```
[the remark this answers is not in the document]
Comment (Grisou13) [reply src/pages/run/run.ts]: I had an error exactly like https://github.com/ionic-team/ionic/issues/11443

And resolved by applying https://github.com/ionic-team/ionic/issues/11443#issuecomment-298823909 this comment

So?

[the remark this answers is not in the document]
Comment (kayoumido) [reply src/pages/run/run.ts]: the comment ....

File: src/pages/runs/runs.ts [hunk 2]
```diff
@@ -30,13 +30,13 @@ export class RunsPage {
...
-    this.loadRuns().subscribe(
-      null,
-      err => err.status != 401 && loader.dismiss(),
-      () => loader.dismiss()
-    )
+    loader.present().then(()=> {
+      this.loadRuns().subscribe(
+        () => loader.dismissAll(),
+        err => err.status != 401 && loader.dismiss(),
+        // () => loader.dismiss()
```
[the remark this answers is not in the document]
Comment (kayoumido) [reply src/pages/runs/runs.ts]: the comment

File: src/app/app.component.ts [hunk 3]
```diff
@@ -14,9 +14,9 @@ export class MyApp {
     platform.ready().then(() => {
       // Set TTL to 12h
       cache.setDefaultTTL(60 * 60 * 12);
-
+      cache.getItem("http://sc-c333-pc08/api/runs?finished=true").then(d => console.log(d)).catch(err => console.log(err))
       // Keep our cached results when device is offline!
-      cache.setOfflineInvalidate(false);
+      // cache.setOfflineInvalidate(false);
```
Comment (kayoumido) [inline src/app/app.component.ts]: If it was old debug code, remove comment

**Pull request merged**